### ISSUE TITLE
Does Entitas suitable for multiplayer games with networking code?

### ISSUE THREAD

**Issue by artfabrique** [sschmid/Entitas-CSharp#830]
Hi,
Does Entitas support FixedUpdate? It's vital for all networking code and state replication/rollback/prediction/other lag compensation methods in a deterministic way. Also maybe it's a good idea to be able to execute systems on "server frame" increments without the need to create a reactive system which reacts to removing and adding a component containing server frame number (sounds like an ultra overkill).

**Comment by FNGgames** [comment]
Yes, you control exactly when your systems are called because you call their Execute methods manually. You can also create multiple sets of systems and execute them separately (some in update, some in fixed, some in late etc)

**Comment by artfabrique** [comment]
@FNGgames Thanks, but where I can find an example, showing how should I organize architecture for a Fixed Update loop?

**Comment by FNGgames** [comment]
Take a look at the game controller in match-one [here](https://github.com/sschmid/Match-One/blob/master/Assets/Scripts/Controllers/GameController.cs). Look how systems are created and how their lifecycle is managed in the unity Start() Update() and OnDestroy() methods. For your needs you'd want to create a second group of systems call fe. `_fixedSystems`, and call `_fixedSystems.Update(); _fixedSystems.Cleanup();` within FixedUpdate() instead of Update().
